apps/server/src/shared/infra/oauth-provider/hydra/hydra.service.spec.ts

```diff
@@ -155,8 +159,22 @@ describe('HydraService', () => {
 
 	describe('Logout Flow', () => {
 		describe('acceptLogoutRequest', () => {
-			it('should throw', () => {
-				expect(() => service.acceptLogoutRequest('')).toThrow(NotImplementedException);
+			it('should make http request', async () => {
+				// Arrange
+				const response: RedirectResponse = { redirect_to: 'redirect_mock' };
+				httpService.request.mockReturnValue(of(createAxiosResponse(response)));
+				const config: AxiosRequestConfig = {
+					method: 'PUT',
+					url: `${hydraUri}/oauth2/auth/requests/logout/accept?logout_challenge=challenge_mock`,
+					headers: { 'X-Forwarded-Proto': 'https' },
+				};
+
+				// Act
+				const redirect: RedirectResponse = await service.acceptLogoutRequest('challenge_mock');
+
+				// Assert
+				expect(httpService.request).toHaveBeenCalledWith(expect.objectContaining(config));
```

Comment:
as usual, I would prefer splitting this into two tests, to follow the "one assert per test" pattern.
(If you are worried about code duplication, the arrange part could be extracted)

Reply:
I think of it more as a "one assert per testCASE". So as long as u have one testcase/usecase that you arrange and act upon, you should expect everything that is to be expected for this exact case. And that can be more than one.